src/server/game/Battlegrounds/Zones/BattlegroundAB.cpp

```diff
@@ -125,7 +122,7 @@ void BattlegroundAB::PostUpdateImpl(uint32 diff)
 
 void BattlegroundAB::StartingEventCloseDoors()
 {
-    for (uint32 obj = BG_AB_OBJECT_BANNER_NEUTRAL; obj < BG_AB_DYNAMIC_NODES_COUNT * BG_AB_OBJECTS_PER_NODE; ++obj)
+    for (uint32 obj = BG_AB_OBJECT_BANNER_NEUTRAL; obj < static_cast<uint8>(BG_AB_DYNAMIC_NODES_COUNT) * BG_AB_OBJECTS_PER_NODE; ++obj)
```

Comment:
Try
```diff
- enum BG_AB_BattlegroundNodes
+ enum BG_AB_BattlegroundNodes : uint8
```
End remove static_cast<uint8>

Reply:
@Winfidonarleyan already tried multiple times :) but it did not work

Reply:
it is very weird because that should set the underlying type since c++11 https://en.cppreference.com/w/cpp/language/enum#:~:text=int-,4),declaration%20for%20a%20scoped%20enumeration%20whose%20underlying%20type%20is%20type,-Each%20enumerator%20becomes